pkg/ddc/jindo/shutdown.go

```diff
@@ -91,13 +92,32 @@ func (e *JindoEngine) releasePorts() (err error) {
 
 // cleanAll cleans up the all
 func (e *JindoEngine) cleanAll() (err error) {
+	count, err := e.cleanupFuse()
+	if err != nil {
+		e.Log.Error(err, "Err in cleaning configMap")
+		return err
+	}
+	e.Log.Info("clean up fuse count", "n", count)
+
+	err = e.cleanConfigmap()
+	if err != nil {
+		e.Log.Error(err, "Err in cleaning configMap")
+		return err
+	}
+	return
+}
+
+// cleanConfigmap cleans up the configmaps, such as:
+// {dataset name}-jindo-values, {dataset name}-jindofs-client-config, {dataset name}-jindofs-config
+func (e *JindoEngine) cleanConfigmap() (err error) {
```

Comment:
I think it's better to use "ConfigMap"  to be consistent with existing method like `kubeclient.DeleteConfigMap`

Reply:
Thanks, I will update according to your suggestions.